src/Classes/Validation.php

```diff
@@ -88,26 +78,10 @@ public static function validateDate(Carbon $date): void
             throw new InvalidDateException('The date must be in the past.');
         }
 
-        if (! self::$earliestPossibleDate) {
-            self::$earliestPossibleDate = Carbon::createFromDate(1999, 1, 4)->startOfDay();
-        }
+        $earliestPossibleDate = Carbon::createFromDate(1999, 1, 4)->startOfDay();
```

Comment:
Does this not tightly couple this to the first-party driver? If I was building out a driver for XE.com, and _could_ reasonably retrieve historical exchange rates, this seems like it would prevent me from even making the call.

Reply:
I'm glad you picked up on this because I wasn't quite sure the best way to tackle this. I guess both of the APIs that are supported at the moment have this limitation. But, as you say, it won't affect all APIs.

I wonder if I should pull this check out of this class because it's specific to just those APIs? Do you have any suggestions where you'd move it to? 🤔

Reply:
I'm leaning towards it not even being something this package enforces. It's bespoke to the API being used, and should probably be handled as a failing API response.

Even if this logic was to be moved elsewhere in the package, it wouldn't prevent exchange rates from failing to exist for currencies that have been created since the start of the API's history. For example, XE.com has API history right back until 1995 for USD, but EUR was only created on 1st January 1999. In that situation, even if the validation limit is defined in config (1995) it'll still fail for _some_ calls.

Reply:
That's actually pretty interesting! Now that you've said this, I think I'm leaning towards removing it like you suggest. 

My main reason for adding it was that it'd prevent API calls that might contribute to the daily API call limits. But I'm guessing it's probably just going to become too complex to maintain when we could just rely on the API to validate the field for us.

I think I'll rip this out in that case, thanks! 😄